libpurecoollink/dyson.py

```diff
@@ -676,11 +725,13 @@ def __init__(self, payload):
         """
         json_message = json.loads(payload)
         data = json_message['data']
-        self._humidity = int(self.__get_field_value(data, 'hact'))
```

Comment:
Exception was thrown here due to `self.__get_field_value(data, 'hact')` returned 'OFF'